azext_edge/edge/providers/check/common.py

```diff
@@ -163,6 +163,9 @@ class DataprocessorDestinationStageType(ListableEnum):
 
 # Check constants
 ALL_NAMESPACES_TARGET = '_all_'
+# when there are runtime resources related to the service but not
+# related to any service resource, use this constant as the resource name
+CORE_SERVICE_RUNTIME_RESOURCE = "coreServiceRuntimeResources"
```

Comment:
Should we call this plural resource as in resources?

Reply:
will update to coreServiceRuntimeResource!